python/pyspark/streaming/flume.py

```diff
@@ -56,6 +56,7 @@ def createStream(ssc, hostname, port,
 
         .. note:: Deprecated in 2.3.0
         """
+        warnings.warn("Deprecated in 2.3.0.", DeprecationWarning)
```

Comment:
for these, could you provide more information? link to the doc on deprecating DStream in python?